### DISCUSSION TITLE
Precise meaning of an explanation pop-up text.

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#11978] [Q&A]
What exactly does the explanation ["When status bar is hidden, this setting will utilize the entirety of screen real state (for your book) and will temporarily overlap status bar and text when shown."](https://hosted.weblate.org/translate/koreader/koreader/en/?checksum=b003b3dc0dd7bdbe) mean?

I understand from usage that it means "When status bar is hidden, this setting will utilize the entirety of screen real state (for your book) and will temporarily overlap text when status bar is shown." Is it correct? If so, perhaps it should be corrected.

Thank you!

**Comment by Commodore64user** [comment]
Thank you for your feedback, I was the one who introduced this change and just want to say that I agree with you here, this is in fact just a bit clearer, i would just add one more thing: When status bar is hidden, this setting will utilize the entirety of screen real state (for your book) and will temporarily overlap _the_ text when status bar is shown.

if @Frenzie is okay with it, we can change it.

**Reply by poire-z** [reply]
I'm more used to read "screen real _estate_" than "screen real state".
Google too https://www.google.com/search?hl=en&num=25&q=%22screen+real+state%22

Is this again a UK vs. US broken English case?

**Reply by Commodore64user** [reply]
> Is this again a UK vs. US broken English case?

no, it is estate. I did not notice that one. I can't believe that got past all of us.
that NEEDS to change too

**Reply by Frenzie** [reply]
When *the* status bar is hidden and *the* status bar is shown.
